docs/_templates/embedded-tutorials.md

```diff
@@ -0,0 +1,38 @@
+# Embedded Tutorial template
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+
+<!-- Copy and use as a basis -->
+
+Start with an introduction to the tutorial. Include the following:
+
+-   What will the reader learn by the end of it?
+-   What hardware, and architecture combination is this tutorial designed for?
+
+## Limitations
+
+Does this platform and architecture combination have any limitations, issues, etc?
```

Comment:
Maybe "Known Limitations"? As a reader, that gives me hope that perhaps one day they will be lifted with a "software update" :)